Activities/Database/ConnectionDialog/UiPath.Data.ConnectionUI.Dialog/SqlConnectionProperties.cs

```diff
@@ -96,6 +109,13 @@ protected override string ToTestString()
             }
             return testString;
         }
+
+        private void ShouldAddEncryptEncrypt()
```

Comment:
`EncryptEncrypt` ? :)

Also, maybe rename to `AddEncryptIfNeeded` since it has side-effects, whereas a method with `Should` in the name does not indicate side-effects.

Reply:
Fixed